src/components/GrabUpdateTable/index.jsx

```diff
@@ -19,55 +21,68 @@ const GrabUpdateTable = () => {
 						<th></th>
 					</tr>
 				</thead>
-				<tbody className="text-center">
-					<tr>
-						<td>25.12.2024</td>
-						<td>12pm - 4pm</td>
-						<td>
-							<span className=" text-xs bg-secondary/50 px-2 rounded-lg ">Inspector</span> <br />
-							<span>Reschedule</span>
-						</td>
-						<td className="font-bold">
-							<Link to={Approutes.grab.inspectionLog}>
-								<Button variant={'primary'} size={'small'} className={'rounded-md'}>
-									View
-								</Button>
-							</Link>
-						</td>
-					</tr>
-					<tr>
-						<td>25.12.2024</td>
-						<td>12pm - 4pm</td>
-						<td>
-							{' '}
-							<span className=" text-xs bg-primary/50 px-2 rounded-lg ">You</span> <br />
-							<span>Reschedule</span>
-						</td>
-						<td className="font-bold">
-							<Link to={Approutes.grab.inspectionLog}>
-								<Button variant={'primary'} size={'small'} className={'rounded-md'}>
-									View
-								</Button>
-							</Link>
-						</td>
-					</tr>
-					<tr>
-						<td>25.12.2024</td>
-						<td>12pm - 4pm</td>
-						<td>
-							<span className=" text-xs bg-primary/50 px-2 rounded-lg ">You</span> <br />
-							<span>Reschedule</span>
-						</td>
-						<td className="font-bold">
-							<Link to={Approutes.grab.inspectionLog}>
-								<Button variant={'primary'} size={'small'} className={'rounded-md'}>
-									View
-								</Button>
-							</Link>
-						</td>
-					</tr>
-				</tbody>
+				{schedulesResult?.schedules.length > 0 ? (
+					<tbody className="text-center">
+						{schedulesResult?.schedules.map((schedule) => {
+							const latest = schedule?.bookings.length - 1;
+							const latestBooking = schedule?.bookings[latest];
+							// console.log('latest', latestBooking);
```

Comment:
This also should be out.
Try use the resolve button when you fix it to help me keep track of what you have fixed.